scheduler/scheduler-server/build.gradle

```diff
@@ -64,8 +64,12 @@ task('functionalTest', type: Test).configure schedulingFunctionalTestConfigurati
 task('performanceTest', type: Test).configure schedulingPerformanceTestConfiguration
 
 task verifyPerformanceChange(type: Test){
-    systemProperty 'pathToStorage', System.getProperty('pathToStorage')
-    systemProperty 'threshold', '0.3'
+    systemProperties << ['threshold': System.getProperty('threshold')]
```

Comment:
Maybe we could give a more precise name for the threshold, like performanceTestThreshold and which could also indicate how the threshold is computed (is it a multiple of a previous run value, an arbitrary number, etc...)

Reply:
You are right